config/samples/receiver_mock_openshift.yaml

```diff
@@ -1251,6 +1255,12 @@ spec:
       enabled: false
       defaultDashboardsEnabled: false
     prometheusOperator:
+      image:
+        repository: public.ecr.aws/sumologic/prometheus-operator
+        tag: v0.43.2-ubi
+      prometheusConfigReloaderImage:
+        repository: public.ecr.aws/sumologic/prometheus-config-reloade
```

Comment:
It seems that it is typo here and it should be `public.ecr.aws/sumologic/prometheus-config-reloader`